index.html

```diff
@@ -105,31 +105,162 @@ <h2>Introduction</h2>
-          <a data-cite="rfc7519#section-4">claim</a> names and header parameters are present, their respective
-          counterpart in the standard <a data-cite="VC-DATA-MODEL#credentials">verifiable credential</a> and
-          <a data-cite="VC-DATA-MODEL#presentations">verifiable presentation</a> MAY be omitted to avoid duplication.
+          This specification describes how to secure media types 
+          expressing Verifiable Credentials and Verifiable Presentations 
+          as described in [[VC-DATA-MODEL]], using JWTs [[RFC7519]].
+        </p>
+        <p class="note">
+          The <code>application/vc+jwt</code> media type described in this specification defines
+          an example of a unidirectional mapping to a base media type defined in 
+          the [[VC-DATA-MODEL]]; see <a href="#example-mapping">Appendix A.4.1</a>.
         </p>
       </section>
+    </section>
 
-      
+
+
+    <section>
+      <h2>Securing JSON</h2>
+      <p class="note">
+        This representation relies on claims registered in the
+        <a href="https://www.iana.org/assignments/jwt/jwt.xhtml#claims">IANA JSON Web Token Claims Registry</a> whenever possible.
+      </p>
+      <p class="note">
+        Production of this representation does not use <code>vc+ld+json</code> as an input.
+      </p>
+      <section>
+        <h2>Credential Header</h2>
+        <p><a data-cite="rfc7519#section-5.1">typ</a> MUST use the media type <code>vc+jwt</code>.</p>
+        <aside
+          class="example"
+          title="credential metadata (decoded JWT header)"
+        >
+          <pre data-include="./examples/vc-2.0/templates/vc+jwt/header.json" data-include-format="json"></pre>
+        </aside>
+      </section>
+      <section>
+        <h2>Credential</h2>
+        <aside
+          class="example"
+          title="credential (decoded JWT payload)"
+        >
+          <pre data-include="./examples/vc-2.0/templates/vc+jwt/claimset.json" data-include-format="json"></pre>
+        </aside>
+        <p class="note">
+          The <code>vc</code> and <code>vp</code> claims MUST NOT be present when the content type header parameter is set to <code>credential-claims-set+json</code>.
+        </p>
+      </section>
+      <section>
+        <h2>Verifiable Credential</h2>
+        <aside
+          class="example"
+          title="verifiable credential"
+        >
+          <p>An encoded verifiable credential (using external proof).</p>
+          <pre data-include="./examples/vc-2.0/vc+jwt.jose" data-include-format="text"></pre>
+        </aside>
+      </section>
+
+    </section>
+
+    <section>
+      <h2>Securing JSON-LD</h2>
+      <section>
+        <h2>With JOSE</h2>
+        <section>
+          <h2>Securing <code>application/vc+ld+json</code> with JOSE</h2>
+          <p>[[rfc7515]] MAY be used to secure this media type.</p>
+          <p><code>typ</code> MUST be <code>vc+ld+jwt</code></p>
```

Comment:
Is the `+ld+jwt` structured syntax suffix going to be registered? If not, how does this align (or not) with the current suffixes draft?